### PR TITLE
Update cambiatus.md

### PR DESCRIPTION
**Pull request by lucca65**
Final version for review

### CODE REVIEW

**Comment by lucca65** [comment]
Folks, please review it as a whole, this will be the final touches before submitting

File: applications/cambiatus.md [hunk 1]
```diff
@@ -280,144 +280,117 @@ This feature will require interactions on:
...
-Documentation
-We will provide both inline documentation of the code and a basic tutorial that explains how a user can use it within their communities. We publish our platform tutorials here: https://medium.com/cambiatus-tutorials
-Feedback gathering and teaching
-We will do live meetings and workshops to showcase basic usage and how existing communities could use this. Meetings will be in English, Spanish and Portuguese, languages that our communities use.
-With feedback we can prioritize next features and adjust as needed
-
-
-Future Plans
+- **Contract:** Actions to authorize Chainlink (or LiquidApps) to our communities to setup automation.
+- **Backend:** No further work will be required if we manage to do this with a solution similar to Chainlink. If not, we will need to develop one similar centralized. It must allow for **triggers** based on our contract ACTIONs:
```
Comment (henriquecbuss) [inline applications/cambiatus.md]:
```suggestion
- **Backend:** No further work will be required if we manage to do this with a solution similar to Chainlink. If not, we will need to develop a similar, centralized one. It must allow for **triggers** based on our contract ACTIONs:
```

File: applications/cambiatus.md [hunk 2]
```diff
@@ -280,144 +280,117 @@ This feature will require interactions on:
...
+  - Add hover for role public view
+  - Add modal for viewing role modal
+  - Add links to modals: profile page, community page, user hover, profile modal
+  - Role cards on the community page
+  - Role wide cards for the new "Our Roles Page"
+
+### Milestone 3 - Gather feedback on the first round of implementations. No code on this milestone, mostly community building, showcasing and learning sessions to use the newly added feature.
+- **Estimated duration:** 2 months
+- **FTE:** 6
+- **Costs:** 10,000.00 USD
```
Comment (henriquecbuss) [inline applications/cambiatus.md:341]: Shouldn't this cost be 20k USD, since it should take 2 months?

File: applications/cambiatus.md [hunk 3]
```diff
@@ -280,144 +280,117 @@ This feature will require interactions on:
...
+  - `upsertobjectives` (when objectives are changed in a community)
+  - `claimaction` (when an user initiates a request for claiming an action)
+  - `verifyclaim` (when other users vote to approve/reject a claim)
+  - `reward` (when admins reward users a certain amount of tokens)
+with the following **Effects**:
+  - `assignrole`
+under available **Conditions**:
+  - Count: a certain number of times a trigger must be activated
+  - Time scope: a limit date like "5 days, 1 month" and so on
+  - Existing roles: a filter that makes sure the affected users have necessary roles
```
Comment (henriquecbuss) [inline applications/cambiatus.md]: The formatting here is weird:

image

I think this should fix it:

```suggestion
  - `reward` (when admins reward users a certain amount of tokens)
  
with the following **Effects**:

  - `assignrole`
  
under available **Conditions**:

  - Count: a certain number of times a trigger must be activated
  - Time scope: a limit date like "5 days, 1 month" and so on
  - Existing roles: a filter that makes sure the affected users have necessary roles
```

File: applications/cambiatus.md [hunk 4]
```diff
@@ -428,14 +401,12 @@ As we progress, we wish to build more refined voting systems, exploring POAPs/NF
...
+## Additional Information
 
 Luiz Hadad, our lead for community building, is an active member of the EOS community since its launch. He worked at EOS Rio leading community building from 2018 until 2019, participated in many EOS conferences and kept an eye on the development of the EOS ecosystem. Since the launch of ENF, we have been willing to participate more in the community. We participated in the Pomelo Grants season 3 and saw on twitter that the grants program was happening. Then Luiz asked Yves La Rose for some information/guidance on the process through telegram, and we got into this stage.
 
 
-Members that contributed financially to the project
+### Members that contributed financially to the project
 Some of our members also help Cambiatus financially by contributing to our Liquidity Pool, here are some of them:
 
 Satisfied Vagabonds LLC
```
Comment (henriquecbuss) [inline applications/cambiatus.md]: Maybe this should be a list? It's kind of confusing right now:

image

**Review by henriquecbuss — APPROVED** [review]
Caught a few small details, but it looks great! 🚀

**Comment by henriquecbuss** [comment]
Oh, also, your CV link is missing (and Luiz's too, but in his section there isn't the "Full CV" text). If we're not going to include your link, maybe we should at least take out the "Full CV"

image

File: applications/cambiatus.md (hunk 2 above)
Comment (lucca65) [reply applications/cambiatus.md:341]: sadly no ): we are capped at 50k total, so its 10k per milestone. We will have to sort it out on those gap months

**Review by henriquecbuss — APPROVED** [review]
✨🚀

**Review by MatheusBuss — APPROVED** [review]
:rocket:

**Comment by juramos-2020** [comment]
HI @lucca65 

Nesse primeiro parágrafo:

> "Cambiatus is already the first step for many people entering the crypto ecosystem and discovering how to use social tokens and collaborate with DAOs. We already have [partner communities](https://www.cambiatus.com/communities) doing this like [Muda](https://www.instagram.com/mudaoutraseconomias/) and [Play4Change](https://www.play4change.io/), empowering people from favelas in Brazil, [Verdes](https://verdesmonteverde.com/) in Costa Rica and [Agelgil](https://www.cambiatus.com/agelgil) in Ethiopia."

Podemos acrescentar a informação da comunidade Paiter Suruí:

"And now we have the newest community called [Paiter Coin](https://www.cambiatus.com/povo-paiter-surui), which is being co-created with the Paiter Suruí Indigenous People, in the quest to involve the people in community actions in the village and have their economy independent with their own currency."

**Comment by juramos-2020** [comment]
Em "Some resources"

> Figma reference with some discussion (in brazilian portuguese)[Figma reference with some discussion (in brazilian portuguese)](https://www.figma.com/file/IYaSqUD58quAZMIhqdi7WA/Roles-%26-Permissions?node-id=0%3A1)
> Slides for the presentation of Role and Permissions[Slides for the presentation of Role and Permissions](https://www.figma.com/file/N724ADHvSImTQkYYnIPOoh/Proposta-ENF_Roles-%26-Permissions)
> 

Por favor trocar o link de "slides for the presentation" para: https://www.figma.com/proto/N724ADHvSImTQkYYnIPOoh/Proposta-ENF_Roles-%26-Permissions?page-id=0%3A1&node-id=26%3A49&viewport=957%2C254%2C0.29&scaling=contain&starting-point-node-id=26%3A49

Também podemos adicionar:

[Mental map](https://miro.com/app/board/uXjVOBYRrXc=/)

Storytelling Presentation in PDF: [Roles&Permissions.pdf](https://drive.google.com/file/d/1ba9zNnsYe2pBF_5lB5VfuGA1ekvz_WNC/view?usp=share_link)
[Mock-ups/designs of any UI components](https://www.figma.com/file/IYaSqUD58quAZMIhqdi7WA/Roles-%26-Permissions?node-id=551%3A7694&t=vO3s9hMAqvU7jIPd-1)
Wireframe in production: [Wireframe Roles and Permissions.pdf](https://drive.google.com/file/d/1yr7H5RMlMqKqgawuBTvsnA9f_1otgQ90/view?usp=share_link)

**Comment by lucca65** [comment]
feito @juramos-2020 pode revisar novamente?

**Comment by juramos-2020** [comment]
Maravilha @lucca65 👌

Por mim ta tudo ok 😉

**Pull request merged**